project_profitability_fields/models/project_project.py

```diff
@@ -0,0 +1,25 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+
+from odoo import api, fields, models
+
+
+class ProjectProject(models.Model):
+    _inherit = "project.project"
+    
+    project_date = fields.Date()

```

Comment:
Why this new field, and not use project built-in `date_start`?

https://github.com/odoo/odoo/blob/e2a684018d7f92994aa5d30ded8f64703254f0fb/addons/project/models/project.py#L340

If the goal is using a different fiels for this addon purpose, we should consider changing its name, because "Project Date" is too generic (and "Project" word is redundant, isn't it?)